src/test/java/it/gov/pagopa/cgn/portal/converter/backoffice/BackofficeApprovedDiscountConverterTest.java

```diff
@@ -18,31 +18,21 @@
 public class BackofficeApprovedDiscountConverterTest {
 
     @Test
-    public void ToDto_TestPendingDiscountToDto_ok() {
-        BackofficeApprovedDiscountConverter converter = getBackofficeAgreementConverter();
-        AgreementEntity agreement = TestUtils.createSampleAgreementEntityWithCommonFields();
-        DiscountEntity discount = TestUtils.createSampleDiscountEntity(agreement);
-        discount.setStaticCode("acode");
-        discount.setState(DiscountStateEnum.TEST_PENDING);
-
-        ApprovedAgreementDiscount discountDto = converter.toDto(discount);
-        Assert.assertNotNull(discountDto);
-        Assert.assertEquals(DiscountState.TEST_PENDING, discountDto.getState());
-        Assert.assertNotNull(discountDto.getStaticCode());
-    }
-
-    @Test
-    public void ToDto_PublishedDiscountToDto_ok() {
+    public void ToDto_TestLandingPageAndReferrerDiscountToDto_ok() {
```

Comment:
Rename this to a more generic name.
```suggestion
    public void ToDto_TestDiscountToDto_ok() {
```